README.md

```diff
@@ -52,7 +62,7 @@ Please download NetMHC using appropriate licensing. Download and installation in
 
```

Comment:
Since NetMHC is a mandatory a pre-requisite, I would suggest that this line be there.